components/definitions/rewardsDonate.ts

```diff
@@ -0,0 +1,46 @@
+declare namespace RewardsDonate {
+  interface State {
+    publisher?: Publisher
+    walletInfo: WalletProperties
+    finished: boolean
+    error: boolean
+  }
+
+  interface ApplicationState {
+    rewardsDonateData: State | undefined
+  }
+
+  interface ComponentProps {
+    actions: any
+    rewardsDonateData: State
+  }
+
+  interface Publisher {
+    publisherKey: string
+    title: string
+    description: string
+    background: string
```

Comment:
Is the publisher background and logo stored locally?

Reply:
no, we get it from publisher API. Example of the url `https://rewards-stg.bravesoftware.com/P5RxjtP44tJnVNeR9nNCLmgX`. In this PR we have both of them disabled and we will fix this in the follow up, because you can't do requests outside from webui